terraform/ec2-test-executors/main.tf

```diff
@@ -0,0 +1,11 @@
+locals {
+  all_instances_matrix_path = "../../versions/strategyMatrix.json"
```

Comment:
We spin up all instances in one shot, but we'll be potentially provisioning them differently using Ansible later.